library/src/test/java/io/constructor/core/ConstructorIoTrackingTest.kt

```diff
@@ -82,6 +82,61 @@ class ConstructorIoTrackingTest {
     private val preferencesHelper = mockk<PreferencesHelper>()
     private val configMemoryHolder = mockk<ConfigMemoryHolder>()
 
+    var emailPii = listOf(
+            "test@test.com",
+            "test-100@test.com",
+            "test.100@test.com",
+            "test@test.com",
+            "test+123@test.info",
+            "test-100@test.net",
+            "test.100@test.com.au",
+            "test@test.io",
+            "test@test.com.com",
+            "test+100@test.com",
+            "test-100@test-test.io"
+    )
+    var phonePii = listOf(
+            "+12363334011",
+            "+1 236 333 4011",
+            "(236)2228542",
+            "(236) 222 8542",
+            "(236)222-8542",
+            "(236) 222-8542",
+            "+420736447763",
+            "+420 736 447 763"
+    )
+    var creditCardPii = listOf(
+            // Sources of example card numbers:
+            // - https://support.bluesnap.com/docs/test-credit-card-numbers
+            // - https://www.paypalobjects.com/en_GB/vhelp/paypalmanager_help/credit_card_numbers.htm
+            "4155279860457", // Visa
```

Comment:
Referring to this [PR](https://github.com/Constructor-io/autocomplete-ui/pull/2966), the first 2 credit cards should not be counted as PII.

Reply:
Totally forgot about this thanks for the reminder!